app/scripts/models/user.js

```diff
@@ -89,12 +107,16 @@ define([
     },
 
     getSignedInAccount: function () {
```

Comment:
The difference between `_getSignedInAccount` and `getSignedInAccount` is not clear from the function names. Is it possible to make the name of `_getSingedInAccount` better reflect its name?